sdk/translation/ai-translation-document-rest/test/public/node/containerHelper.ts

```diff
@@ -95,9 +73,9 @@ async function createContainer(
   containerName: string,
   documents?: TestDocument[],
 ): Promise<ContainerClient> {
-  const blobServiceClient: BlobServiceClient = BlobServiceClient.fromConnectionString(
-    env.DOCUMENT_TRANSLATION_CONNECTION_STRING as string,
-  );
+  const storageName = env.DOCUMENT_TRANSLATION_STORAGE_NAME as string;
```

Comment:
Nit: consider either validating that the env var exists or using the `!` operator here

Option a:

```ts
function getEnvVar(name: string) { 
  const val = env[name];
  if (val === undefined) { throw new Error("....") }
  return val;
}

...

const storageName = getEnvVar("DOCUMENT_TRANSLATION_STORAGE_NAME");
```

Option B:

```ts
  const storageName = env.DOCUMENT_TRANSLATION_STORAGE_NAME!
```

Reply:
Thanks for the suggestion. Updated